docs/sources/administration/configuration.md

```diff
@@ -202,15 +202,14 @@ Or redirect port 80 to the Grafana port using:
 $ sudo iptables -t nat -A PREROUTING -p tcp --dport 80 -j REDIRECT --to-port 3000
 ```
 
-Another way is put a webserver like Nginx or Apache in front of Grafana and have them proxy requests to Grafana.
+Another way is to put a web server like Nginx or Apache in front of Grafana and have them proxy requests to Grafana.
 
 ### domain
 
-This setting is only used in as a part of the `root_url` setting (see below). Important if you use GitHub or Google OAuth.
```

Comment:
I'm not clear why this line was deleted - was this intentional?

Reply:
Thanks for catching that deletion. I am also unclear on this.

Reply:
Do we have definitive guidance on using "webserver" versus "web server" in our style guide? If not, we should probably add instructions to use "web server" and not "webserver".

Reply:
@achatterjee-grafana was that comment intended for [line 205](https://github.com/grafana/grafana/pull/30695/files#diff-d0b4774dfc86c7e56111aa3e75307f80bccde523195641b239ce17051880ff2aR205) instead?

Reply:
Lime 205 - "web server".

Reply:
Currently, there is no entry for that in the style guide, and I too vote to add it as web server. Rationale being that web page is not written webpage, and web browser is not webbrowser. Will handle that in a subsequent PR.

Reply:
@osg-grafana @achatterjee-grafana - can you comment on ln205 about that instead? this comment thread is for ln209 😉